charmm_templates/NAMD.conf

```diff
@@ -18,7 +18,7 @@ firsttimestep      0
 #############################################################
 # Input
 paraTypeCharmm	    on
-parameters          /home/Programs/EntropyMaxima/params/charmm27.ff/par_all27_prot_na.prm 
+parameters          /code/em/params/charmm27.ff/par_all27_prot_na.prm
```

Comment:
all of the python files should reference the files in the installed packages folder. All charmm files are referencing the repository which is mounted in the code folder.